### PR TITLE
Adding support for Rain and Wind sensors

### PR DESCRIPTION
**Pull request by jarlebh**
This pull request also adds support for refresh command for the Telldus live bridge

### CODE REVIEW

File: addons/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/handler/live/TelldusLiveDeviceController.java [hunk 1]
```diff
@@ -273,43 +275,54 @@ public void onRequest(TellstickDeviceEvent newDevices) {
...
+        }
+        return obj;
     }
 
-    private void logResponse(String uri, Response resp) {
-        if (resp != null) {
-            logger.warn("Request [" + uri + "] Response:" + resp.getResponseBody());
+    private void logResponse(String uri, Exception e) {
+        if (e != null) {
+            logger.warn("Request [" + uri + "] Failure:" + e.getMessage());
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/handler/live/TelldusLiveDeviceController.java]: could you change this to parameterized logging?

File: addons/binding/org.openhab.binding.tellstick/ESH-INF/thing/sensor.xml [hunk 2]
```diff
@@ -17,29 +17,39 @@
...
-				<description>The name of the device (manually entered)</description>
-				<required>false</required>
-			</parameter>
-			<parameter name="deviceId" type="text">
-				<label>DeviceID</label>
-				<description>This function returns the unique id of a device with a specific index.</description>
-				<required>true</required>
-			</parameter>
-		</config-description>
+		<config-description-ref uri="thing-type:tellstick:sensor-config"/>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.tellstick/ESH-INF/thing/sensor.xml]: there seems to be a tabs/spaces mix in here now

File: addons/binding/org.openhab.binding.tellstick/ESH-INF/config/config.xml [hunk 3]
```diff
@@ -17,5 +17,26 @@
 			<required>false</required>
 		</parameter>
 	</config-description>
-
+    <config-description uri="thing-type:tellstick:sensor-config">
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.tellstick/ESH-INF/config/config.xml]: there seems to be a tabs/spaces mix in here now

**Comment by jarlebh** [comment]
I replaced all tabs with 4 spaces and the logging change.

**Comment by kaikreuzer** [comment]
Thanks, you are quick.

**Pull request merged**